### PR TITLE
Update words that can be translated

### PR DESCRIPTION
**Pull request by MDVAurelien**
Update the words that can be translated but they will have to be injected into the PO file.

### CODE REVIEW

**Comment by erinix** [comment]
This doesn't build, looking at your PR I see that you did things like {{Configure | translate}}, this needs to be {{"Configure" | translate}}, otherwise it thinks you are passing a variable named Configure into the translate pipe

**Comment by erinix** [comment]
or also {{'Configure' | translate}} when inside of double quotes

**Comment by erinix** [comment]
still not quite right

**Comment by MDVAurelien** [comment]
Yes sorry I'm going to correct, I went a little too fast!

**Comment by MDVAurelien** [comment]
I still have a error but I don't see where it is...

File: src/app/core/components/widgets/widgetnic/widgetnic.component.html [hunk 1]
```diff
@@ -24,18 +24,18 @@
 
               <button mat-icon-button class="" (click)="router.navigate(['network/interfaces/edit/'+nicState.name])">
                 <mat-icon role="img"
-                          matTooltip="Configure {{nicState.name}}"
+                          matTooltip="{{'Configure' | translate}} {{nicState.name}}"
                   matTooltipPosition="above"
-                  aria-label="Configure Interface">
+                  aria-label="{{'Configure Interface' | translate}}">
```
Comment (erinix) [inline src/app/core/components/widgets/widgetnic/widgetnic.component.html]: the build is complaining about these aria-labels, try removing the translates from these

File: src/app/core/components/widgets/widgetnic/widgetnic.component.html [hunk 2]
```diff
@@ -24,18 +24,18 @@
...
                 </mat-icon>
               </button>
 
               <button mat-icon-button class="" (click)="router.navigate(['reportsdashboard/network'])">
                 <mat-icon role="img"
-                  matTooltip="Network Reports"
+                  matTooltip="{{'Network Reports' | translate}}"
                   matTooltipPosition="above"
-                  aria-label="Network Reports">
+                  aria-label="{{'Network Reports' | translate}}">
```
Comment (erinix) [inline src/app/core/components/widgets/widgetnic/widgetnic.component.html]: this one too

**Comment by MDVAurelien** [comment]
alléluia :)

**Pull request merged**